vm/kvm-unit-tests/runtest.sh

```diff
@@ -165,6 +165,38 @@ cat test.log | tee -a $OUTPUTFILE
 # check for any new failures
 grep FAIL test.log >> failures.txt
 
+cat >/etc/yum.repos.d/rhel8-advvirt.repo <<EOF
```

Comment:
Hi @mcondotta thanks for the PR :-) do you think the test should disable the repo after running the test to properly cleanup since this test is run in conjunction with other tests ?

Reply:
Hello @rasibley thank you very much for the review.
I've added the cleanup code to remove the Repo and downgrade the QEMU version.

Reply:
Thanks looks good!